### ISSUE TITLE
Support node.js 10+

### ISSUE THREAD

**Issue by ocavue** [VerbalExpressions/JSVerbalExpressions#195]
JSVerbalExpressions only supports node version "^9.2.0", which means "10.0.0 > version >= 9.2.0", based on it's package.json

```
$ yarn add verbal-expressions                                                                                                                                         
yarn add v1.16.0
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error verbal-expressions@1.0.0: The engine "node" is incompatible with this module. Expected version "^9.2.0". Got "12.4.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

https://github.com/VerbalExpressions/JSVerbalExpressions/blob/78febfc8c0b8c76ee1899fe5880e4bda0ae01754/package.json#L48-L50

node.js version 9.x [has become unsupported](https://nodejs.org/en/about/releases/). JSVerbalExpressions should support a newer node version.

**Comment by ionutvmi** [comment]
This has been resolved in version 1.0.1

Can you please check and confirm that everything is good on your side ?

**Comment by ocavue** [comment]
Look good on my side
